Atomic.md

```diff
@@ -0,0 +1,618 @@
+
+This is an account that has at least one inner transaction.
+
+#### 2.5.2. `SigningPubKey` and `Signature`
+
+These fields are included if the account is signing with a single signature (as opposed to multi-sign). They sign the `Flags` and `TxnIDs` fields.
+
+#### 2.5.3. `Signers`
+
+This field is included if the account is signing with multi-sign (as opposed to a single signature). It operates equivalently to the [`Signers` field](https://xrpl.org/docs/references/protocol/transactions/common-fields/#signers-field) used in standard transaction multi-sign. This field holds the signatures for the `Flags` and `TxnIDs` fields.
+
+### 2.6. Metadata
+
+The inner transactions will be committed separately to the ledger and will therefore have separate metadata. This is to ensure better backwards compatibility for legacy systems, so they can support `Atomic` transactions without needing any changes to their systems.
+
+For example, a ledger that only has one `Atomic` transaction containing 2 inner transactions would look like this:
+```
+[
+  OuterTransaction,
+  InnerTransaction1,
+  InnerTransaction2
+]
+```
+
+#### 2.6.1. Outer Transactions
+
+Each outer transaction will only contain the metadata for its sequence and fee processing, not for the inner transaction processing.
+
+There will also be a list of which transactions were actually processed, which is useful for the `ONLYONE` and `BATCH` atomicity types, since those may only process a subset of transactions, and for debugging with all atomicity types. This section will be called `AtomicExecutions`.
+
+It will contain a list of objects that have the following fields for every transaction that is processed (successfully or unsuccessfully):
+
+|FieldName | Required? | JSON Type | Internal Type |
+|:---------|:-----------|:---------------|:------------|
+|`TransactionHash`|✔️|`string`|`STUInt256`|
+|`TransactionResult`|✔️|`string`|`STUInt8`|
+
+Some important things to note:
+* It is possible that all transactions will not be included in this list. For example, when using the `ONLYONE` atomicity type, if the first transaction succeeds, then the rest of the transactions will not even be processed. 
+* Transactions will only be included in the ledger if their result code is `tesSUCCESS` _and_ if the outer transaction has a result code of `tesSUCCESS`. For example, the inner transaction might have a result code of `tesSUCCESS` without being included in the ledger if the `ALL` atomicity type is used, but one of the transaction fails.
+
+#### 2.6.2. Inner Transactions
+
+Each inner transaction will contain the metadata for its own processing. Only the inner transactions that were actually committed to the ledger will be included. This makes it easier for legacy systems to still be able to process `Atomic` transactions as if they were normal.
+
+There will also be a pointer back to the parent outer transaction (`parent_atomic`), for ease of development (similar to the `nftoken_id` field).
```

Comment:
Yes, the transaction ID. IMO it'd still be nice to have the transaction ID for ease of fetching - should be easy enough for the RPCs to find it from `AtomicTxn`, so it would probably add less load to the RPCs than `nftoken_id`.